promql/promqltest/testdata/native_histograms.test

```diff
@@ -1163,6 +1163,51 @@ eval_warn range from 0 to 12m step 6m metric{series="2"} - ignoring (series) met
 
 clear
 
+# Test incompatible schemas with comparison binary operators
+load 6m
+  metric{series="1"} {{schema:-53 sum:1 count:1 custom_values:[2] buckets:[1]}} {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}}
+  metric{series="2"} {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}} {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}}
```

Comment:
Why not do the following and then simplify the expressions below to `metric_1 == metric_2`?

```suggestion
  metric_1 {{schema:-53 sum:1 count:1 custom_values:[2] buckets:[1]}} {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}}
  metric_2 {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}} {{schema:-53 sum:1 count:1 custom_values:[5 10] buckets:[1]}}
```